src/System.Net.HttpListener/tests/HttpResponseStreamTests.cs

```diff
@@ -478,8 +478,7 @@ public void Dispose()
 
                 // Disconnect the Socket from the HttpListener.
                 client.Close();
-                GC.Collect();
-                GC.WaitForPendingFinalizers();
+                Thread.Sleep(2000);
```

Comment:
This isn't reliable.  We either need to find a way to coordinate between the client and server to ensure the proper behavior, or delete the test.